db/primary_migrate/20221129175844_add_inherited_proofing_proofed_to_proofing_component.rb

```diff
@@ -0,0 +1,5 @@
+class AddInheritedProofingProofedToProofingComponent < ActiveRecord::Migration[7.0]
+  def change
+    add_column :proofing_components, :inherited_proofing_proofed, :boolean, default: false
```

Comment:
As I understand it most of the team we are loading a row of proofing components by the user_id, is there a case where we'd query by this column?

Reply:
Cool, thank you. I'm trying to get a better sense of the tradeoffs here. If there's a specific query/report that would make use of an index on the column, could you share it? We have a good number of reports that run unindexed queries and it's fine since it's not super time sensitive, and I wonder if that may be the case here as well.

Reply:
@mitchellhenke actually, now that you mention it, I am not sure. I need to fine out if ProofingComponents for a user is deleted after they are copied to the user profile after the user is proofed. On top of that, CloudWatch uses the analytics log files from what I understand, so we may not actually need this. I'll check if there's anything else that may need this index that I am not aware.

Reply:
@mitchellhenke yea, it looks like there is no need for that index ATM. Will remove.

Reply:
Removed.